model/place.go

```diff
@@ -23,13 +23,15 @@ type Uint32Slice []uint32
 
 // Place holds information about a place
 type Place struct {
-	ID               uint32      `json:"id" dynamodbav:"pk"`
+	ID               uint32      `json:"id" dynamodbav:"pk,string"`
+	Type             string      `json:"-" dynamodbav:"sk"`
+	AltSort          string      `json:"-" dynamodbav:"altSort"`
 	Name             string      `json:"name"`
-	FullName         string      `json:"fullName"`
-	AltNames         StringSlice `json:"altNames"`
-	Types            StringSlice `json:"types"`
+	FullName         string      `json:"fullName" dynamodbav:"fullName"`
+	AltNames         StringSlice `json:"altNames" dynamodbav:"altNames"`
+	Types            StringSlice `json:"types" dynamodbav:"types"`
 	LocatedInID      uint32      `json:"locatedInId"`
```

Comment:
just curious, why not have a dynamodbav annotation on LocatedInID, or CountryID?

Reply:
It defaults to using the names in the `json` annotation, so the better question is why FullName, AltNames and Types have dynamodbav annotations. I'll look at that.